root/static/scripts/edit/components/RemoveButton.js

```diff
@@ -13,7 +13,7 @@ class RemoveButton extends React.Component {
     return (
       <button
         className="nobutton icon remove-item"
-        onClick={this.props.callback}
+        onClick={this.props.onButtonRemoved}
```

Comment:
This one's confusing, since no button is being removed. I'd keep it simple and name this `onClick`.